packages/go/openapi/src/schemas/model.environment-access-control.yaml

```
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: Apache-2.0
+
+type: object
+properties:
+  user_id:
+    type: string
+    format: uuid
+    description: The user UUID that associates the access control to a specific user
+    readOnly: true
+  environment_id:
+    type: string
+    description: The environment id which a user has been granted access to
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)